install/jaxrs/bin/ts.jte

```diff
@@ -234,7 +219,7 @@ optional.tech.packages.to.ignore=jakarta.xml.bind
 #    4 - deploy and run only
 #
 # @harness.socket.retry.count - denotes the number of time we should
-#          attempt to create a server socket when initializing a test
+#          attempt to create a server socket when intilizing a test
```

Comment:
Please keep correct spelling of `initializing`

Reply:
For making JDK 11 as base, I have replaced ts.jte with ts.jte.jdk11. issue is from https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/install/jaxrs/bin/ts.jte.jdk11, rectified same.